Use .new on BodySerializer setter examples

ugexe opened the pull request:
See: https://github.com/croservices/cro-http/blob/21e4076d1b00e8eb29d03eee38d21cced74e5d2a/t/http-server.t#L280

Altai-man commented:
Thanks for your proposal!

Although, I feel need to note that these calls work with type objects as well as with instances/objects, so basically these `.new` calls in tests are optional(or, maybe, redundant).

As a matter of micro-optimization, I'd vote for removing these `.new` from the test rather than enforce it through documentation.

ugexe commented:
I thought this failed for me until I added the `.new`. It must have been something else I changed because I just retried without the `.new` after your comment and it worked.

Altai-man commented:
Did https://github.com/croservices/cro-http/commit/529c417390d0f8e43060092c903facdb474e1ad3 to prevent accidental misunderstanding. And we should probably document that too...
